pkg/cmd/root/root.go

```diff
@@ -0,0 +1,23 @@
+package root
+
+import (
+	"github.com/spf13/cobra"
+
+	initCmd "github.com/Schrodinger-Hat/Daje/pkg/cmd/init"
```

Comment:
Why did you use the remote path for the package init and not the local relative path?